source/interact-data/raw-queries.txt

```diff
@@ -0,0 +1,383 @@
+.. _django-raw-queries:
+
+============================
+Perform Raw Database Queries
+============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: number, amount, estimation, code example
+
+Overview
+---------
+
+In this guide, you can learn how to use {+django-odm+} to run
+raw queries on your MongoDB database. Raw queries allow you to
+query the database by using MongoDB's aggregation pipeline syntax
+rather than Django methods.
+
+The Django QuerySet API provides a ``QuerySet.raw()`` method, which allows
+you to perform raw SQL queries on relational databases. {+django-odm+}
+does not support the ``raw()`` method. Instead, the ODM provides the
+``QuerySet.raw_aggregate()`` method, which you can use to send instructions
+to the database in pipeline stages.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``Movie`` and ``Theater`` models, which
+represent collections in the ``sample_mflix`` database from the :atlas:`Atlas sample datasets </sample-data>`.
+The model classes have the following definitions:
+
+.. code-block:: python
+
+    from django.db import models
+    from django_mongodb_backend.fields import EmbeddedModelField, ArrayField
+    from django_mongodb_backend.managers import MongoManager
+
+    class Movie(models.Model):
+        title = models.CharField(max_length=200)
+        plot = models.TextField(null=True)
+        runtime = models.IntegerField(default=0)
+        released = models.DateTimeField("release date", null=True)
+        awards = EmbeddedModelField(Award)
+        genres = ArrayField(models.CharField(max_length=100), blank=True)
+        objects = MongoManager()
+
+        class Meta:
+            db_table = "movies"
+
+        def __str__(self):
+            return self.title
+    
+    class Theater(models.Model):
+        theaterId = models.IntegerField(default=0)
+        objects = MongoManager()
+
+        class Meta:
+            db_table = "theaters"
+        
+        def __str__(self):
+            return self.title
```

Comment:
I think this should probably be moved to an includes file so we can reuse it

Applies to all code examples

Reply:
Moving this to the [docs-django repo](https://github.com/mongodb/docs-django/pull/1) now - I'll put the code in a separate file there!